css/components/notification.postcss

```diff
@@ -3,14 +3,55 @@
 \*----------------------------------------*/
 
 .notification {
-  @apply flex items-start;
+  @apply flex items-center;
+  @apply max-w-lg;
   @apply p-4 rounded;
+  @apply text-sm;
 
   &.is-error {
```

Comment:
I see two options here to keep consistency with the css names:
1. like the other existing components we write `.notification .notification--name-of-the-modifier` (check badge.postcss file for example)
2. we keep `.is-success` and similar but we make them generic

Reply:
I would say lets stick to notification--name-of-the-modifier for the moment. Its more clear and maybe there will be cases where we need to overwrite the normal behavior. WDYT?